src/main/apiClient.ts

```diff
@@ -619,11 +619,11 @@ export default class ApiClient {
     }
 
     return this.http
-      .post<CredentialCreationOptionsSerialized>('/webauthn/registration-options', { body, accessToken })
+      .post<RegistrationOptions>('/webauthn/registration-options', { body, accessToken })
       .then(response => {
-        const options = encodePublicKeyCredentialCreationOptions(response.publicKey)
+        const publicKey = encodePublicKeyCredentialCreationOptions(response.options.publicKey)
 
-        return navigator.credentials.create({ publicKey: options })
+        return navigator.credentials.create({ publicKey })
       })
       .then(credentials => {
         if (!credentials || credentials.type !== publicKeyCredentialType) {
```

Comment:
For the record there is a typo in the next line (in the throw)
crendentials -> credentials